src/Base/RestAPI/ItemFields/SeoPress.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Adds portal url field to the output.
+ */
+
+namespace OWC\OpenPub\Base\RestAPI\ItemFields;
+
+use OWC\OpenPub\Base\Models\Item;
+use OWC\OpenPub\Base\Support\CreatesFields;
+use WP_Post;
+
+class SeoPress extends CreatesFields
+{
+    /**
+     * The condition for the creator.
+     *
+     * @return callable
+     */
+    protected function condition(): callable
+    {
+        return function () {
+            return true;
+        };
+    }
+
+    /**
+     * Create the portal seopress fields for a given post.
+     *
+     * @param WP_Post $post
+     *
+     * @return string
+     */
+    public function create(WP_Post $post): array
+    {
+        $seoMeta = $this->plugin->config->get('seopress.meta');
+
+        $postMeta = \get_post_meta($post->ID, '', true);
+
+        $sharedMeta = $this->filteredMetadata($seoMeta, $postMeta);
+
+        return $this->mappedMetadata($sharedMeta);
+    }
+
+    public function filteredMetadata(array $seometa, array $postmeta): array
```

Comment:
Ik zou proberen om een betere naam te verzinnen voor deze functie zodat het direct duidelijk is wat hier gebeurd.